### PR TITLE
Add Fee Reversal

### PR DESCRIPTION
**Pull request by dev-dennis-guye**
Add support for fee reversal. See [Fee Reversal](https://docs.unit.co/fees/#reverse-fee)
See: https://github.com/unit-finance/unit-node-sdk/pull/322

### CODE REVIEW

File: types/fee.ts [hunk 1]
```diff
@@ -82,4 +82,71 @@ export interface CreateFeeRequest {
...
+    type: "fee reversal"
+    /**
+     * Identifier of the fee resource.
+     */
+    id: string
+    attributes: {
+        /**
+         * Description of the fee.
+         */
+        description: number
```
Comment (andre-moves) [inline types/fee.ts]: description is a number? I would have expected string. 🤷

**Comment by qoire** [comment]
Okay, that documentation is a little confusing, the payload doesn't match the description of the fields. But assuming the example payload is correct, it looks good.

**Review by andre-moves — APPROVED** [review]

**Review by qoire — APPROVED** [review]

File: types/fee.ts (hunk 1 above)
Comment (qoire) [reply types/fee.ts]: Oops I missed that

Comment (dev-dennis-guye) [reply types/fee.ts]: I'll fixed that one :D

**Comment by dev-dennis-guye** [comment]
> Okay, that documentation is a little confusing, the payload doesn't match the description of the fields. But assuming the example payload is correct, it looks good.

@qoire  I am aware of the discrepancies and I've reached out to unit concerning this. 

https://gonuco.slack.com/archives/C01KX2895GW/p1681832575384339

**Pull request merged**